bindings/dotnetcore/dotnet-core-binding/Microsoft.Azure.Devices.Gateway/Gateway.cs

```diff
@@ -0,0 +1,84 @@
+﻿using System;
+using System.Runtime.InteropServices;
+
+namespace Microsoft.Azure.Devices.Gateway
+{
+    /// <summary>
+    ///  Wrapper for Azure IoT Gateway SDK. 
+    /// </summary>
+    public class Gateway
+    {
+        [DllImport(
+            "gateway",
+            CharSet = CharSet.Ansi,
+            EntryPoint = "Gateway_CreateFromJson",
+            CallingConvention = CallingConvention.Cdecl
+        )]
+        private static extern IntPtr CreateFromJsonInternal(string file_path);
+
+
+        /// <summary>
```

Comment:
I removed the detailed JSON from the Header. I agree it's too much for a header documentation.

---
In reply to: [104802834](https://github.com/Azure/azure-iot-gateway-sdk/pull/151#discussion_r104802834) [](ancestors = 104802834)